src/networks.json

```diff
@@ -735,13 +735,13 @@
     "network": "mainnet",
     "multicall": "0x024f0041b76B598c2A0a75004F8447FaF67BD004",
     "rpc": [
-      "https://rpc.coredao.org/"
+      "https://rpcar.coredao.org/"
     ],
     "explorer": {
       "url": "https://scan.coredao.org"
     },
     "start": 853908,
-    "logo": "ipfs://QmVctLQ44vhkwejja9DDjDYUdYgVRBEWs242mhd95SeM5q"
+    "logo": "ipfs://QmdUgiGaD6fGjhb9x1NvfccvEBBFq2YHSm9yb8FNhpwHkW"
```

Comment:
Hi @leeyanbtcs Make sure to pin this IPFS. It doesn't resolve when i go to https://ipfs.io/ipfs/QmdUgiGaD6fGjhb9x1NvfccvEBBFq2YHSm9yb8FNhpwHkW

I have update RPC on our Internal list, proposals should be working now :)

Reply:
updated you can check https://ipfs.io/ipfs/bafkreigjv5yb7uhlrryzib7j2f73nnwqan2tmfnwjdu26vkk365fyesoiu